Feliz/Styles.fs

```diff
@@ -89,410 +94,652 @@ type style =
     ///
     /// **Tip**: This property is useful if you want to create overlapping elements.
     static member inline zIndex(value: int) = Interop.mkStyle "zIndex" value
+    /// Sets the margin area on all four sides of an element. It is a shorthand for margin-top, margin-right, 
+    /// margin-bottom, and margin-left.
     static member inline margin(value: int) = Interop.mkStyle "margin" value
+    /// Sets the margin area on all four sides of an element. It is a shorthand for margin-top, margin-right, 
+    /// margin-bottom, and margin-left.
     static member inline margin(top: int, right: int) =
         Interop.mkStyle "margin" (
             (unbox<string> top) + "px " +
             (unbox<string> right) + "px"
         )
-
+    /// Sets the margin area on all four sides of an element. It is a shorthand for margin-top, margin-right, 
+    /// margin-bottom, and margin-left.
     static member inline margin(top: ICssUnit, right: int) =
         Interop.mkStyle "margin" (
             (unbox<string> top) + " " +
             (unbox<string>right) + "px"
         )
-
+    /// Sets the margin area on all four sides of an element. It is a shorthand for margin-top, margin-right, 
+    /// margin-bottom, and margin-left.
```

Comment:
These overloads set the margin for specific sides of the element not all four of them: the overload in this case sets the margin for the top and right side. Same holds for the other overloads, can you please change them?